AndroidSDKCore/src/main/java/com/leanplum/internal/FeatureFlagManager.java

```diff
@@ -0,0 +1,19 @@
+package com.leanplum.internal;
+
+import android.support.annotation.VisibleForTesting;
+
+import java.util.HashSet;
+
+public class FeatureFlagManager {
+    public static final FeatureFlagManager INSTANCE = new FeatureFlagManager();
```

Comment:
If you use static INSTANCE, should you make the constructor `private`? It looks like you're using `Singleton`.

Reply:
Thanks for the review Toan! 👍